scripts/scripts.js

```diff
@@ -334,6 +333,17 @@ function decorateOfferLinks(main) {
   });
 }
 
+/**
+ * loads a block named 'v2-footer' into footer
+ */
+function loadFooter(footer) {
+  if (footer) {
+    const footerBlock = buildBlock('v2-footer', '');
```

Comment:
Now you load the new footer every time - for the old pages too. You should check if the page is a new design probably. Or maybe we have the agreement that the footer and navigation are new for all of the pages?

Reply:
Yes we are loading new footer to all the pages old and new